### DISCUSSION TITLE
Where is the recommended spot for sound_player for basic target sounds?

### DISCUSSION THREAD

**Discussion by ohmbrew** [missionpinball/missionpinball#17] [Q&A]
I've implemented the recommended mode layering found here:

https://docs.missionpinball.org/en/latest/game_design/mode_layering.html

There, it recommends placing background music and sounds in the "Field" mode. However, when player qualifies for "Missions", the Field mode is stopped as the Mission mode is started. Seems I would need to duplicate all my sounds in a sound_player in each mission config file. Seems redundant...should I place all my common target sounds (that will always play no matter what field/mission/wizard mode is running) lower in the layers, like in the Base mode? And then have mission specific sounds (e.g. the mission's background music) be included in each mission specific mode?

**Comment by ohmbrew** [comment]
I tested this - moved some standard sounds that play every time a target is hit - to my base mode (which always runs during a player's turn) and they still play, as it should. Just curious why background music and sounds are listed just under the field mode, since that goes away when missions or wizard modes start.

**Comment by JAMESCARDONA-PINBALL** [comment]
one correction to what you said is that base mode is pretty much always running as long as a game is being played. It is not running during attract or service mode. Also not that base mode shuts down at the end of ball and then immediately restarts at the start of a new ball so there is a few milliseconds there when it is not running. I put all of the yaml hooks that I want available at all times in my global config folder so they can be called by any mode at any time.
